includes/news/mail/in-the-news.php

```diff
@@ -18,22 +18,39 @@
 					<ul style="list-style: none; padding-left: 0; margin: 0;">
 					<?php
 					foreach( $external as $story ) :
+						$story_source = ( !empty( $story->source_name ) ) ? $story->source_name : $story->source;
 					?>
 						<li style="padding-bottom: 30px;" align="left">
 							<table>
 								<tbody>
-								<tr>
-									<td style="padding-bottom: 4px;">
-										<a class="montserratsemibold" style="font-family: Helvetica, Arial, sans-serif; font-size: 18px; color: #000000; font-weight: 500;" href="<?php echo $story->url; ?>">
-											<?php echo $story->link_text; ?>
-										</a>
-									</td>
-								</tr>
-								<tr><td style="padding-top: 5px;">
-									<a class="montserratlight" style="font-family: Helvetica, Arial, sans-serif; font-size: 12px; color: #757575; text-decoration: none; text-transform: uppercase;" href="<?php echo $story->url; ?>">
-										<?php echo $story->source; ?>
-									</a>
-								</td></tr>
+									<tr>
+									<?php if( !empty( $story->source_image ) ) : ?>
+										<td style="padding-right: 10px;">
+											<a href="<?php echo $story->url; ?>">
+												<img class="responsiveimg" border="0" width="50" style="border:none;" src="<?php echo $story->source_image; ?>">
+											</a>
+										</td>
+									<?php endif; ?>
```

Comment:
What do you think about adding in an else statement here that creates a table cell that contains a lighter/grey placeholder icon for stories that do not have source icons/images added? That allows for all of the news stories' titles to be lined up. I created this example with the external story icon, but there might be a better icon for news:
![gmucf-example](https://user-images.githubusercontent.com/21267784/48579659-1c9fe680-e8eb-11e8-8ca5-46172998665b.jpg)